### ISSUE TITLE
Fine-tuning dataset for emotion2vec+ base

### ISSUE THREAD

**Issue by Choddeok** [ddlBoJack/emotion2vec#46]
Hello, I am curious about the fine-tuning dataset for emotion2vec+ base. Can I assume that the fine-tuning was done using the downstream tasks datasets mentioned in 'Table 1: The datasets at a glance for emotion2vec pre-training and downstream tasks'?

**Comment by ddlBoJack** [comment]
Hi, emotion2vec+ seed is fine-tuned with academic speech emotion data from [EmoBox](https://github.com/emo-box/EmoBox) and emotion2vec+ base is ine-tuned with filtered large-scale pseudo-labeled data labeled from emotion2vec+ seed.

**Comment by Choddeok** [comment]
Thank you for your response. I would like to know if there are any plans to release the model checkpoints that follow the 'Table 1: The datasets at a glance for emotion2vec pre-training and downstream tasks' setup?

**Comment by ddlBoJack** [comment]
Do you mean the downstream models?

**Comment by zhoujiani0102** [comment]
> Do you mean the downstream models?你指的是下游模型吗？

Hello! I would like to know which funasr pipeline checkpoints should I use for downstream tasks?

**Comment by jeeyung** [comment]
Hello! I was wondering if you can share the training data from EmoBox you used for fine-tuning emotino2vec+ seed. Specifically, how did you split your data into train/val/test?
